templates/cli/lib/questions.js.twig

```diff
@@ -515,9 +584,35 @@ const questionsMfaChallenge = [
     }
 ];
 
+const questionGetEndpoint = [
+    {
+        type: "input",
+        name: "endpoint",
+        message: "Enter the endpoint of your {{ spec.title|caseUcfirst }} server",
+        default: "http://localhost/v1",
```

Comment:
We should use cloud as default

Reply:
Cloud is default.

This question is available only when the self-hosted flag is passed. Would it still make sense to change it to Appwrite cloud?